.github/workflows/ci.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby_version: ["2.7", "3.0", "3.1", "3.2"]
+        ruby_version: ["3.0", "3.1", "3.2"]
```

Comment:
Can we add 3.3 while we're here as well?

Reply:
👍🏻